src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -80,7 +90,7 @@
    * Path to the data (datastore, blobstore, etc.) associated with the application. (default: None)
    */
   @Parameter(alias = "devserver.storagePath", property = "app.devserver.storagePath")
-  protected String storagePath;
+  protected File storagePath;
```

Comment:
@loosebazooka

Does maven automatically convert the string to file? Will this have any effect on the end user configs?